### DISCUSSION TITLE
Tabby and GNS3 Telnet Sessions

### DISCUSSION THREAD

**Discussion by iam-thehelp** [Eugeny/tabby#6660] [Q&A]
*Tabby Version: 1.0.181
GNS3 Version: 2.2.33.1*

Hello,

I use GNS3 and wanted to know if there is a way to use Tabby as the Custom Console application.

I looked at the help option for Tabby, but could not find any additional information for Telnet connections.

Would anyone know if this is possible at this time with Tabby and GNS3 variables?

<img width="522" alt="GNS3_Custom_Console" src="https://user-images.githubusercontent.com/101709761/175313031-7e1f69d9-8917-4bad-aafc-42202a4cefda.png">

**Comment by eantowne** [comment]
I am actually looking into this right now. I have managed to get Tabby to open with a specific profile, but I cannot find any documentation about opening new tabs from cli, nor passing parameters for something like a telnet session.

If I figure it out, I will post here, along with submitting to GNS3 devs for inclusion.
